backend/src/data/repositories/user-repository.ts

```diff
@@ -38,10 +43,21 @@ class User {
     });
   }
 
-  public getByType(type: UserType): Promise<IUserWithPermissions[]> {
+  public getByType(type: UserType, filter: IDoctorFiltrationPayload): Promise<IUserWithPermissions[]> {
     if (type === UserType.DOCTOR) {
+      const where = { type };
+
+      if (filter.doctorName)
```

Comment:
Check **B12** - https://github.com/BinaryStudioAcademy/bsa-winter-2021-healthcare/blob/development/quality-criteria.md#formatting-and-code-styling

Reply:
Do something like:

![image](https://user-images.githubusercontent.com/60396171/112103847-0a1cd280-8bb3-11eb-94dc-21d5d57ad537.png)

Don't forget, that array(even if it's empty) is alway `true`